IBX-8916: Content update in User documentation - cd. (#314)
ibexa/documentation-user: docs/content_management/schedule_publishing.md

![Publish Later button in the menu](img/publish_later.png "Publish Later button in the menu")

A **Future publication settings** window is displayed.

> Choose a date and time and for the content publication.

Reviewer: ```suggestion
Choose a date and time for the content publication.
```

## Date-based hiding

> When your content item is published, you can schedule a date and time and the content is hidden at that time.

Reviewer: ```suggestion
When your content item is published, you can schedule a date and time for the content to become hidden.
```
I feel like the current sentence still works better with "will be", I've tried rewriting this to avoid this word.
